desc/magnetic_fields.py

```diff
@@ -452,12 +457,21 @@ def __init__(self, R, phi, Z, BR, Bphi, BZ, method="cubic", extrap=False, period
     def _approx_derivs(self, Bi):
         tempdict = {}
         tempdict["fx"] = _approx_df(self._R, Bi, self._method, 0)
-        tempdict["fy"] = _approx_df(self._phi, Bi, self._method, 1)
         tempdict["fz"] = _approx_df(self._Z, Bi, self._method, 2)
-        tempdict["fxy"] = _approx_df(self._phi, tempdict["fx"], self._method, 1)
         tempdict["fxz"] = _approx_df(self._Z, tempdict["fx"], self._method, 2)
-        tempdict["fyz"] = _approx_df(self._Z, tempdict["fy"], self._method, 2)
-        tempdict["fxyz"] = _approx_df(self._Z, tempdict["fxy"], self._method, 2)
+        if self._axisym:
+            tempdict["fy"] = jnp.zeros_like(tempdict["fx"])
+            tempdict["fxy"] = jnp.zeros_like(tempdict["fx"])
+            tempdict["fxy"] = jnp.zeros_like(tempdict["fx"])
```

Comment:
Is this supposed to be `tempdict["fyz"]`?

Reply:
yeah, I left in that typo to see if you were really reading all the PRs

Reply:
Caught me out then. I swear I was reading but glazed over that line completely